processor/src/main/java/org/mapstruct/ap/internal/model/BeanMappingMethod.java

```diff
@@ -608,12 +612,11 @@ private void reportErrorForUnmappedTargetPropertiesIfRequired() {
             // fetch settings from element to implement
             ReportingPolicyPrism unmappedTargetPolicy = getUnmappedTargetPolicy();
 
-            //we handle forged methods differently than the usual source ones. in
-            if ( method instanceof ForgedMethod ) {
+            //we handle automapping forged methods differently than the usual source ones. in
+            if ( method instanceof ForgedMethod && ( (ForgedMethod) method ).isAutoMapping() ) {
```

Comment:
I think we should only do this for forged automappings and do the normal reporting for the other forged methods. Especially with the addition to ignore unmapped target properties for our own ForgedMethods (which needs to be revised)